README.md

```diff
@@ -220,8 +220,66 @@ within it. This allows for any unexpected state data of the user from hanging ar
 # Deploying the app
 ## If you decide to deploy to Heroku:
 * [Heroku Command Line Interface](https://devcenter.heroku.com/categories/command-line)
+* If you have HTTPS enabled (requires a paid dyno), be sure to enable `config.force_ssl = true` on line 47 of `config/environments/production.rb`
+
 ## If you decide to use Docker:
-* [Docker Desktop](https://www.docker.com/products/docker-desktop) to use the docker-compose steps listed above
+
+_Note_: These instructions assume you are working with shell access to a system with Docker and docker-compose available (such as a Digital Ocean Docker image).
+
+_Note_: These instructions assume you already have a domain name registered and pointed it at the
+IP address of your server.
+
+1. You'll need to get the code on the server. The recommended way to do this is with a [GitHub Deploy Key](https://developer.github.com/v3/guides/managing-deploy-keys/). Deploy Keys are just SSH keys that are tied to a project, instead of being tied to a user, which means that they can persist independently of the project's membership. You can use [GitHub's guide on generating SSH keys to generate your deploy key](https://help.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent).
+2. After you've generated your deploy key and added it to your project, clone the project.
+3. Create a Caddyfile. It can live anywhere (Mutual Aid Tompkins has ours at `~/.config/Caddyfile`), and it should look like this, replacing `your.domain.name` with your organization's desired domain name:
+
+   ```
+   your.domain.name {
+     header {
+       Strict-Transport-Security "max-age=31536000; includeSubdomains; preload"
+     }
+
+     root * /config/public
+
+     # Serve static files (like assets and packs) with the file_server
+     @static_files {
+       file
+     }
+     file_server @static_files
+
+     # Anything that's not a static file, proxy to puma/rails
+     @backend {
+       not file
+     }
+     reverse_proxy @backend http://app:3000
+   }
+   ```
+
+4. Navigate to the `docker/production` directory within your clone of the repo. You should see a file named `docker-compose.yml`.
+5. Run
+   ```bash
+   docker-compose run app '/usr/local/bin/rake' secret \
+    | echo "SECRET_KEY_BASE=$(tail -1 -)" > .env
+   ```
+   This will generate a secret token and add it to a file named `.env`, which will be available when
+   `docker-compose` starts.
+6. Generate (using a password manager) a password for the Postgres database, and append it to the `.env` file. Your `.env` file should look something like:
+   ```bash
+   SECRET_KEY_BASE=<A fairly long string of numbers>
+   POSTGRES_PASSWORD=<your postgres password>
+   ```
```

Comment:
Some thoughts about `.env` files.
1. As of #532 , the main repo `.env` file is no longer empty and has all our default values, so line 262 above might need to append instead of write.
1. Something feels off about having to mess with `.env` files within the container. Keep in mind that [dotenv](https://github.com/bkeepers/dotenv#can-i-use-dotenv-in-production) isn't designed to even load in a typical production setting. Is there an alternate way to set environment variables with docker? That said, i really don't know docker well, so may be completely missing many things!